api/v1/sriovnetwork_types.go

```diff
@@ -42,6 +42,9 @@ type SriovNetworkSpec struct {
 	// +kubebuilder:validation:Minimum=0
 	// Maximum tx rate, in Mbps, for the VF. Defaults to 0 (no rate limiting)
 	MaxTxRate *int `json:"maxTxRate,omitempty"`
+	// MetaPluginsConfig configuration to be used in order to chain metaplugins to the sriov interface returned
+	// by the operator.
+	MetaPluginsConfig string `json:"metaplugins,omitempty"`
```

Comment:
Shall we use `runtime.RawExtension` to store the MetaPluginsConfig? It will be useful if we need to manipulate the content of the meta plugin config.

Reply:
Good point. I used string to be consistent with the ipam field, but I agree makes more sense to use RawExtension.
Will change

Reply:
I want to change the `ipam` field to `runtime.RawExtension` as well. But I am not quite sure whether it will break the backward compatibility or not. We can address that in separate PR later.

Reply:
Question: if we go with the RawExtension, the user will need to put the yaml-ified version of the cni config, right?

For example 

```
metaplugins: |
    { "type": "firewall", "backend": "iptables" }
```

becomes something like

```
  metaplugins:
    plugins:
      - type: firewall
        backend: iptables

```

Am I missing something? If this is the case, I feel this is a bit worse for the user, as all the meta cni plugin examples are in the json form and they'll need to do the mental translation.
Thoughts?

Reply:
Yeah. That is a problem. I agree it's better to keep using string here.

Reply:
yes, use string, or is rawMessage an option?

Reply:
I think rawMessage suffers of the same issue: we must declare the field as object, which eventually gets marshalled.
So in that case the user will have to convert the yaml to json.

If we want to leave the json format, I think `string` is the only option.